osu.Game/Screens/Edit/EditorScreenWithTimeline.cs

```diff
@@ -99,6 +99,11 @@ private void load([CanBeNull] BindableBeatDivisor beatDivisor)
                     }
                 },
             };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
```

Comment:
What's this change about? Triggering content load at `LoadComplete()` seems worse than in `load()`, unless it's something about not having the right beatmap to access or the like?

Reply:
Precisely that. I don't remember the exact exception but they definitely weren't getting the type cached by this screen.